pkg/cli/helm/helm.go

```diff
@@ -78,6 +79,20 @@ func locateChartFile(dirPath string) (string, error) {
 	return filepath.Join(dirPath, files[0].Name()), nil
 }
 
+type anonymousTransport struct {
+}
+
+func (t *anonymousTransport) RoundTrip(req *http.Request) (*http.Response, error) {
+	req.Header.Del("Authorization")
```

Comment:
@sk593 @willdavsmith Deleting Authorization header doesn't work with custom transport. I got 401 instead of 403. Looks like GHCR still needs some authorization header even for anonymous pull.